receiver/iisreceiver/config.go

```diff
@@ -0,0 +1,27 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package iisreceiver // import "github.com/open-telemetry/opentelemetry-collector-contrib/receiver/iisreceiver"
+
+import (
+	"go.opentelemetry.io/collector/receiver/scraperhelper"
+
+	"github.com/open-telemetry/opentelemetry-collector-contrib/receiver/iisreceiver/internal/metadata"
+)
+
+// Config defines configuration for simple prometheus receiver.
+type Config struct {
+	scraperhelper.ScraperControllerSettings `mapstructure:",squash"`
+	metricSettings                          metadata.MetricsSettings
```

Comment:
Let's keep it consistent with other scrapers?
```suggestion
	Metrics                          metadata.MetricsSettings `mapstructure:"metrics"`
```